gradio/reload.py

```diff
@@ -0,0 +1,54 @@
+"""
+
+Contains the functions that run when `gradio` is called from the command line. Specifically, allows
+
+$ gradio app.py, to run app.py in user reload mode where any changes in the app.py reload the demo.
+$ gradio-dev app.py, to run app.py in developer reload mode where any changes in the Gradio library reloads the demo.
+$ gradio app.py my_demo, to use variable names other than "demo"
+$ gradio-dev app.py my_demo, to use variable names other than "demo"
+"""
+import os
+import sys
+
+from gradio import networking
+
+
+def run_in_reload_mode(mode: str):
+    args = sys.argv[1:]
+    if len(args) == 0:
+        raise ValueError("No file specified.")
+    if len(args) == 1:
+        demo_name = "demo"
+    else:
+        demo_name = args[1]
+
+    original_path = args[0]
+    path = original_path.replace("/", ".")
```

Comment:
Needs support for windows paths as well. For example, I get an error if I try:

`$ gradio .\demo\calculator\run.py`

```
  File "c:\users\islam\appdata\local\programs\python\python39\lib\site-packages\uvicorn\importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "c:\users\islam\appdata\local\programs\python\python39\lib\importlib\__init__.py", line 122, in import_module
    raise TypeError(msg.format(name))
TypeError: the 'package' argument is required to perform a relative import for '.\\demo\\calculator\\run'
```

Also seems like leading "." might be necessary on Windows? I get an error without it:

`$ gradio demo\calculator\run.py`

```
ERROR:    Error loading ASGI app. Could not import module "demo\calculator\run".
```

The easiest way might be to convert Windows path to *NIX format.

Reply:
Hmm, when I tried in windows autocomplete was providing demo/calculator/run.py paths and it was working.
The problem with NX format is the char's were different than how it looks in the string and that's why I was not able to replace characters, will take a look.

Reply:
Done!